### PR TITLE
feat(integration): add strava-web-oauth integration

### PR DESCRIPTION
**Pull request by findachin**
## Describe your changes
- Added configuration for strava-web in providers.yaml
- Added strava-web to existing strava docs-v2 .
- Added strava-web SVG file in template-logos folder.

## Issue ticket number and link
NA

## Test
This provider has been used successfully in local development to connect to strava-web oauth after authorizing using Nango.
image
image

The documentation update was reviewed using npm run docs and verifying on localhost.
image

### CODE REVIEW

File: packages/webapp/public/images/template-logos/strava-web.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+strava.svg
```
Comment (TBonnin) [inline packages/webapp/public/images/template-logos/strava-web.svg:1]: Thank you for your contribution. the svg looks malformed. Can you fix it?

Comment (findachin) [reply packages/webapp/public/images/template-logos/strava-web.svg:1]: image
I have added the image sym link similar to other implementations. 

Similar to stripe-app-sandbox.svg
image

Please suggest the required change.

Comment (TBonnin) [reply packages/webapp/public/images/template-logos/strava-web.svg:1]: My bad. I read it as it was the content of the svg not a symlink.

**Review by TBonnin — APPROVED** [review]

File: packages/webapp/public/images/template-logos/strava-web.svg (hunk 1 above)
Comment (achincrosshatch) [reply packages/webapp/public/images/template-logos/strava-web.svg:1]: No problem and thank you for your swift approval.

**Pull request merged**